tests/core/fsck-on-boot/task.yaml

```diff
@@ -54,7 +87,7 @@ execute: |
       # Use offset 65 as FAT32 kicks in for devices larger than 32MB
       printf "\x01" > one
       tests.cleanup defer rm -f one
-      dd if=one of=/dev/sda2 seek=65 bs=1 count=1 conv=notrunc
+      dd if="/dev/disk/by-label/$LABEL" of=dirty skip=65 bs=1 count=1 conv=notrunc
```

Comment:
this is now wrong, it was
```
dd \
  if=one \
  of=/dev/... \
  seek=65 \
  ...
```
and it has now become:
```
dd \
  if=/dev/...
  of=dirty \
  skip=65 \
  ...
```

perhaps this line was mixed up with the one below?

Reply:
yes, thanks for checking

Reply:
@anonymouse64 updated